patches/0013-mali-support-building-against-4.15.patch

```diff
@@ -1,17 +1,22 @@
-From 5bc1f96c6e9b8927e3020fdfe58b2d99fa42bcb3 Mon Sep 17 00:00:00 2001
-From: John Stultz <john.stultz@linaro.org>
-Date: Thu, 16 Nov 2017 13:42:23 -0800
-Subject: MALI: v4.15-rc build fix
+From 1fd429daef6dc5f4b2efbf032caacb473f027364 Mon Sep 17 00:00:00 2001
+From: Giulio Benetti <giulio.benetti@micronovasrl.com>
+Date: Tue, 5 Dec 2017 22:32:31 +0100
+Subject: [PATCH 1/1] mali support building against 4.15-rc
+
+Used new timer api provided by <linux/timer.h>
 
 Signed-off-by: John Stultz <john.stultz@linaro.org>
+Signed-off-by: Giulio Benetti <giulio.benetti@micronovasrl.com>
 ---
- src/devicedrv/mali/common/mali_control_timer.c  | 2 +-
- src/devicedrv/mali/common/mali_group.c          | 8 +++++---
- src/devicedrv/mali/common/mali_osk_types.h      | 8 +++++++-
- src/devicedrv/mali/linux/mali_memory_os_alloc.c | 2 +-
- src/devicedrv/mali/linux/mali_osk_timers.c      | 8 +-------
- 5 files changed, 15 insertions(+), 13 deletions(-)
+ src/devicedrv/mali/common/mali_control_timer.c  |  2 +-
+ src/devicedrv/mali/common/mali_group.c          |  8 +++++---
+ src/devicedrv/mali/common/mali_osk_types.h      |  8 +++++++-
+ src/devicedrv/mali/linux/mali_memory_os_alloc.c |  2 +-
+ src/devicedrv/mali/linux/mali_osk_timers.c      | 16 ++++++++++------
+ 5 files changed, 24 insertions(+), 12 deletions(-)
```

Comment:
Or that part.

Reply:
@mripard This depends on the fact I've added timer_setup function in case kernel_version < 4.14